src/stun_listener.erl

```diff
@@ -199,7 +199,7 @@ accept(ListenSocket, Opts) ->
         {ok, Socket} ->
             case {inet:peername(Socket), inet:sockname(Socket)} of
                 {{ok, {PeerAddr, PeerPort}}, {ok, {Addr, Port}}} ->
-                    ?LOG_DEBUG("Accepting connection: ~s -> ~s",
+                    ?LOG_INFO("Accepting connection: ~s -> ~s",
```

Comment:
Are you sure this will change anything? 
https://github.com/jellyfish-dev/fake_turn/blob/258681812647730daf4ad0258f1adc4d6b51e11f/include/stun_logger.hrl#L28
https://github.com/jellyfish-dev/fake_turn/blob/258681812647730daf4ad0258f1adc4d6b51e11f/include/stun_logger.hrl#L30